vllm/v1/core/sched/scheduler.py

```diff
@@ -505,14 +495,32 @@ def schedule(self) -> SchedulerOutput:
                     # chunked prefill has to be enabled explicitly to allow
                     # pooling requests to be chunked
                     if (
-                        not self.scheduler_config.enable_chunked_prefill
+                        not self.scheduler_config.chunked_prefill_enabled
                         and num_new_tokens > token_budget
                     ):
                         self.waiting.pop_request()
                         skipped_waiting_requests.prepend_request(request)
                         continue
 
-                    num_new_tokens = min(num_new_tokens, token_budget)
+                    if (envs.VLLM_USE_LIGHTER_MAMBA_CACHE
```

Comment:
make this a util function to avoid code duplication of first prefill / chunked prefill?